package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Accepts a directory full of json-schema files and produces associated validators and TypeScript interfaces. Great for validating server side with NodeJS and improving TypeScript client side dev.",
   "main": "dist/index.js",
   "files": [
-    "dist/index.js"
+    "dist"
```

Comment:
this code is not bundled, all the files in the dist folder are needed when packaging for NPM